web/src/routes/admin/pipelines/index.svelte

```diff
@@ -78,11 +78,11 @@
   <title>Pipelines</title>
 </svelte:head>
 
-<main class="h-full overflow-y-auto">
-  <div class="container px-6 mx-auto grid">
+<main class="h-full overflow-hidden">
+  <div class="container px-6 mx-auto grid overflow-hidden">
     <h2 class="my-6 text-2xl font-semibold text-gray-700 dark:text-gray-200">Pipelines</h2>
     <!-- New Table -->
-    <div class="w-screen h-screen overflow-hidden rounded-lg shadow-xs">
+    <div class="w-full h-screen overflow-hidden rounded-lg shadow-xs">
```

Comment:
Going from 100vw to 100%, interesting. Looks better?

Reply:
100vw = the actual screen size
100% = the full width of the current component

```mermaid
  graph TD;
      A100vw-->F(B50vw);
      A100vw-->S(C50vw);
      F(B50vw)-->F1(C25vw);
      F(B50vw)-->S1(D25vw);
```

Now, for C and D 100% = 25vw